Gemfile

```diff
@@ -11,6 +11,8 @@ gem 'sprockets-rails'
 # Use postgresql as the database for Active Record
 gem 'pg', '~> 1.1'
 
+gem 'rubocop', '>= 1.0', '< 2.0'
+
```

Comment:
- You have done well so far, but there are some issues with your project
- I currently cannot fetch greetings from your API because I get this CORS error when opening your project. Please add the cors gem to your project, or you can simply follow this [guide](https://medium.com/@chris.lty07/configuring-for-cors-with-ruby-on-rails-part-ii-fb5399a80d64) on how to have it installed. Please fix this 👍 👍 
![gawergaer](https://github.com/zbahati/rails-react-webpack/assets/73607512/dfdb9e63-0963-461c-80ec-cbf1d137f017)
- In your `greetingSlice.js` file, please remove the `console.log` statement in your `createAsyncThunk` method. It is not good practice to log anything to the console. Please fix this 🙏 
![gqw4gw4g](https://github.com/zbahati/rails-react-webpack/assets/73607512/eceedcc8-8bc4-4fa0-8df6-29efcd4da770)